distmesh/_distmesh2d.py

```diff
@@ -29,7 +29,7 @@
 # Functions
 #-----------------------------------------------------------------------------
 
-def distmesh2d(fd, fh, h0, bbox, pfix=None, fig='gcf'):
+def distmesh2d(fd, fh, h0, bbox, pfix=None, fig='gcf', **kwargs):
```

Comment:
OK with @popersson sign-off I think we can add this. Any reason you chose to use `kwargs` over providing them? As far as documenting the function, providing the defaults in the signature is much more helpful for Python tooling (i.e. I see no compelling reason to use `**kwargs` here other than the sheer number of them).

Reply:
Yes sounds great - I agree that sometimes it is useful to modify these, in particular `dptol`.

Reply:
Yes, the sheer number of the arguments seemed overwhelming so I thought the signature would be more readable this way (I followed what is used in other libraries, e.g. scikit here: https://github.com/scikit-image/scikit-image/blob/master/skimage/measure/_structural_similarity.py#L12). Happy to move them to the signature, if that's preferred.

Reply:
Yes I would say it's preferred not to use `kwargs`. (Looking at the `scikit` example it's not clear why they do it / what they gain from it since they still have to document them in the docstring.)